Apollo/Components/CopyOffset.xaml

```diff
@@ -5,9 +5,12 @@
              DetachedFromVisualTree="Unloaded">
   
   <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Stretch" MaxHeight="223">
-    <StackPanel Orientation="Horizontal" VerticalAlignment="Center" Margin="3 0 0 0">
-      <Components:MoveDial x:Name="Offset" Title="Offset" />
-      <Components:Remove VerticalAlignment="Top" Margin="0 5 0 0" Clicked="Offset_Remove" />
+    <StackPanel Orientation="Vertical" VerticalAlignment="Center" Spacing="5">
```

Comment:
Use a Grid here with each element vertically aligned in its own half. See how Delay and Switch are done